jetstream/metrics/__init__.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" All logic around adding metrics to the Prometheus client registry """
+
+import prometheus_client
+import shortuuid
+
+instance_uuid = shortuuid.uuid()
+
+# Metrics we care to observe
+prefill_backlog = prometheus_client.Gauge(
+    name="jetstream_prefill_backlog_size",
+    documentation="Size of prefill queue",
+    labelnames=["uuid"],
```

Comment:
Seems it's better if you could get the pod id or container id in GKE environment, so that you could map the metric to the server container or pod. I feel the existing GKE projects may have some references?

Reply:
We could set it to something like `os.environ['HOSTNAME']`, are we certain that this is always defined? If not should we keep as is or fallback to another value?